docs/configuration/dataValue.md

```diff
@@ -0,0 +1,40 @@
+# DataValue
```

Comment:
Names are not consistants: "datavalue" vs "valuelabel" and actually I'm not sure what should be the plugin name: "valuelabel", "valuelabels", "datalabel" or "datalabels"? I think I would go for "datalabels"